drivers/usbc/tcpc/Kconfig.tcpc_stm32

```diff
@@ -0,0 +1,13 @@
+# USBC STM32 TCPC device configuration options
+
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+config USBC_TCPC_STM32
+	bool "USBC TCPC device controller driver for STM32 devices"
+	depends on SOC_FAMILY_STM32
```

Comment:
nit: may be removed (line below implies this)

Reply:
Done